api/prometheus/api.go

```diff
@@ -42,6 +43,11 @@ const (
 	epSeries      = "/series"
 )
 
+const (
+	// ContextBearerTokenKey is used to create a context from Beaer Token
+	ContextBearerTokenKey = "Bearer"
```

Comment:
Context keys should be unexported types to avoid polluting the global namespace.  See: https://golang.org/pkg/context/#WithValue